database/migrations/2019_12_14_000001_create_personal_access_tokens_table.php

```diff
@@ -14,12 +14,13 @@ class CreatePersonalAccessTokensTable extends Migration
     public function up()
     {
         Schema::create('personal_access_tokens', function (Blueprint $table) {
-            $table->bigIncrements('id');
-            $table->uuidMorphs('tokenable');
+            $table->id();
```

Comment:
Next time don't do this by changing a migration, but always do this by creating a new migration. This so forwards compatibility can be maintained. (For now it doesn't matter as we're still in the development state)

Reply:
They did this xD

Reply:
It's our problem if it breaks on our end ;P
They probably changed it in their initializing project. So you only get this code on project creation.